backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/payment/JsonPaymentService.java

```diff
@@ -144,4 +243,64 @@ private OrgId retrieveOrg(@RequestBody @NonNull final JsonInboundPaymentInfo jso
 		// TODO it would be nice here if we would also use orgID when searching for the bpartner, since an ExternalId may belong to multiple users from different orgs.
```

Comment:
...not just nice..TODO: 
* add OrgId as a parameter OR
* make sure it fails if there are multiple bpartners with the same externalId OR
* don't use bpartnerPriceListServicesFacade but something else (which then filters by org)

Reply:
added OrgId as param.